crates/webidl-tests/global.js

```diff
@@ -0,0 +1,4 @@
+global.global_no_args = () => 3;
```

Comment:
This should probably be

```js
global.global_no_args = function () {
    assert(this == global);
    return 3;
}
```

just to make sure we properly bind the global method to its global object.